index.html

```diff
@@ -52,114 +52,86 @@ <h1>Personal Profile</h1>
 				</div>
 				
 				<div class="sectionContent">
-					<p>I am a former registered nurse still licensed in both California and Texas. I am looking to make a career change into software development.</p>
+					<p>I am a former registered nurse looking to make a career change into software development.</p>
 				</div>
 			</article>
 			<div class="clear"></div>
 		</section>
         
         <section>
 			<div class="sectionTitle">
-				<h1>Education</h1>
+				<h1>Projects</h1>
 			</div>
 			
 			<div class="sectionContent">
                 <article>
-                    <h2>EDX Free Online Courses</h2>
-                    <p class="subDetails">July 2015 - July 2016</p>
-                    <p>Completed two Introduction to Computer Science courses.</p>
+                    <h2>BikeRacks</h2>
+                    <p class="subDetails">Interactive map</p>
+                    <p><ul>
+                        <li>Helps users find safe bike racks in their area</li>
+                        <li>Interactive voting and filtering of bike racks</li>
+                        <li>Tech used: JavaScript, jQuery, Flask, Python, Firebase, Leaflet</li>
+                    </ul></p>
                 </article>
 				<article>
-					<h2>Austin Community College, Austin, TX</h2>
-					<p class="subDetails">January 2014 - May 2014</p>
-					<p>Earned a Nursing Skills Review Certificate.</p>
+					<h2>CheckItCalendar</h2>
+					  <p class="subDetails">Productivity tracker</p>

```

Comment:
"Productivity tracker" is very broad and yet somehow still doesn't capture it.

Maybe these subdetails would be better as the sales pitch?